Actioner integration (#2307)
DataDog/integrations-extras: actioner/README.md

# Actioner

## Overview

Actioner is a no-code app builder that allows you to build your own apps for Slack and workflow automations without requiring any coding. This integration uses OAuth to connect your account with Actioner but does not publish or fetch data. With this integration, you can create workflows in Actioner using the Datadog API.

## Setup

After you install the integration, click the **Connect Account** button in Datadog to be redirected to your Actioner account to set up the OAuth2 connection.

> ## Data Collected

Reviewer: ```suggestion

## Data Collected
```

# Actioner

## Overview

Alternative way to connect your Datadog account with Actioner.

## Setup

Follow the steps below to establish connection between Actioner and Datadog accounts.

1. Log in to your Actioner account.
2. Go to profile settings page from the bottom-left corner. Switch to my connections tab.

> 3. Click on the "Add connection", select Datadog connection. Proceed with the "Connect"

Reviewer: The Oauth process should be kicked off from Datadog. Please update the the instructions to start at Datadog and the steps the customer will take

## Data Collected

### Metrics

Actioner does not publish any metrics.

### Service Checks

Actioner does not include any service checks.

### Events

Actioner does not include any events.

## Support

> Need help? Contact [Datadog support][1].

Reviewer: This should be your support contact for the integration

# Actioner

## Overview

> Actioner is a no-code app builder that lets you build your own apps for Slack and workflow automations without coding
> anything.
> This integration establishes necessary connection which lets you use Datadog API Actions in Actioner. Actioner does not
> publish or fetch data by itself, but a workflow in which Datadog API calls are made can be constructed.

Reviewer 2: ```suggestion
Actioner is a no-code app builder that allows you to build your own apps for Slack and workflow automations without requiring any coding.
This integration establishes the required connection to use Datadog API Actions in Actioner. Actioner does not
publish or fetch data, however, you can create a workflow to accomplish this using Datadog API calls.
```

Reviewer 2: Quick question. When you refer to "workflow automations", are you referring to the Datadog product or to generic "workflows"? If you're referring to the Datadog product, can we change "workflow automations" to "Workflow Automation"?

Author: I'm referring generic workflows which may include Datadog API calls.

## Setup

> Once you install the integration, click the "Connect Account" button in Datadog. This should redirect you to Actioner
> account,
> where you can establish OAuth2 connection.

Reviewer 2: ```suggestion
After you install the integration, click the **Connect Account** button in Datadog to be redirected to your Actioner account to set up the OAuth2 connection.
```
